x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/kibana/disable_internal_collection_instructions.js

```diff
@@ -0,0 +1,221 @@
+              }}
+            />
+          </p>
+        </EuiText>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+          xpack.monitoring.kibana.collection.enabled: false
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.kibanaInstructions.disable_internal_collection.note"
+              defaultMessage="Leave the {config} set to its default value ({defaultValue})."
+              values={{
+                config: (
+                  <Monospace>xpack.monitoring.enabled</Monospace>
+                ),
+                defaultValue: (
+                  <Monospace>true</Monospace>
+                )
+              }}
+            />
+          </p>
+        </EuiText>
+        {restartWarning}
+      </Fragment>
+    )
+  };
+
+  let migrationStatusStep = null;
+  if (!product || !product.isFullyMigrated) {
+    let status = null;
+    if (hasCheckedStatus) {
+      let lastInternallyCollectedMessage = '';
+      // It is possible that, during the migration steps, products are not reporting
+      // monitoring data for a period of time outside the window of our server-side check
+      // and this is most likely temporary so we want to be defensive and not error out
+      // and hopefully wait for the next check and this state will be self-corrected.
+      if (product) {
+        const lastInternallyCollectedTimestamp = product.lastInternallyCollectedTimestamp || product.lastTimestamp;
+        const secondsSinceLastInternalCollectionLabel =
+          formatTimestampToDuration(lastInternallyCollectedTimestamp, CALCULATE_DURATION_SINCE);
+        lastInternallyCollectedMessage = (<FormattedMessage
+          id="xpack.monitoring.metricbeatMigration.kibanaInstructions.disable_internal_collection.partiallyMigratedStatusDescription"
+          defaultMessage="Last internal collection occurred {secondsSinceLastInternalCollectionLabel} ago."
+          values={{
+            secondsSinceLastInternalCollectionLabel,
+          }}
+        />);
+      }
+
+      status = (
+        <Fragment>
+          <EuiSpacer size="m"/>
+          <EuiCallOut
+            size="s"
+            color="warning"
+            title={i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.partiallyMigratedStatusTitle',
+              {
+                defaultMessage: `We still see data coming from internal collection of Kibana.
+                Note that it can take up to {secondsAgo} seconds to detect.
+                We will continuously check every {timePeriod} seconds in the background.`,
+                values: {
+                  secondsAgo: meta.secondsAgo,
+                  timePeriod: autoCheckIntervalInMs / 1000,
+                }
+              }
+            )}
+          >
+            <EuiText>{lastInternallyCollectedMessage}</EuiText>
```

Comment:
EuiCallOut already has an `EuiText` wrapper inside it. Just change these to `p` tags.
```suggestion
            <p>{lastInternallyCollectedMessage}</p>
```

Reply:
<img width="941" alt="Screen Shot 2019-06-14 at 10 05 46 AM" src="https://user-images.githubusercontent.com/56682/59514949-03b0e100-8e8c-11e9-821b-3e0360ff08f2.png">

Reply:
Wrong screenshot?

Reply:
Yes, my bad

<img width="869" alt="Screen Shot 2019-06-14 at 10 06 40 AM" src="https://user-images.githubusercontent.com/56682/59515001-23e0a000-8e8c-11e9-83a5-b6cc2bf56392.png">